src/app/admin/product-form/product-form.component.html

```diff
@@ -87,11 +87,15 @@
   </div>
   <div class="col-md-6">
     <div class="card" style="width: 18rem;">
-      <img [src]="imageUrl.value" class="card-img-top" />
+      <img
+        [src]="product.imageUrl"
+        *ngIf="product.imageUrl"
+        class="card-img-top"
+      />
       <div class="card-body">
-        <h5 class="card-title">{{ title.value }}</h5>
+        <h5 class="card-title">{{ product.title }}</h5>
         <p class="card-text">
-          {{ price.value | currency: 'USD':true }}
+          {{ product.price | currency: 'USD':true }}
```

Comment:
For angular 5+ the currency pipe has changed. For this case, just remove the :true and you should be good. If you want a list of all the arguments it takes just google angular currency pipe

Reply:
Cool didn't realize that.  Good catch.